includes/rest-api.php

```diff
@@ -623,10 +623,11 @@ function check_post_types_permissions() {
  * @return \WP_REST_Response|\WP_Error
  */
 function get_pull_content_list( $request ) {
-	$args = [
+	$post_type = ! empty( $request['post_type'] ) ? $request['post_type'] : array( 'post', 'page' );
```

Comment:
The default is set to `post` in `get_pull_content_list_args()` so it's unlikely this will ever be set to `[post, page]`.

I included it back when I was rewriting this as a safety check just in case someone messes with the URL. 

Was the intent with this change to default to the View All view? For internal connections Distributor offers a View All view but defaults to posts only, so I think it's best to match that for the time being.

Reply:
@peterwilsoncc  Replaced it to show only the `post` by default.

The intent with this change was to get the list of post type that needs to be listed or else default to `post`. But, we are not defaulting the post type filter to `View All`. It is still `Post`